docs/machine-learning/tutorials/taxi-fare.md

```diff
@@ -128,32 +128,32 @@ var pipeline = new LearningPipeline();
 Next, load your data into the pipeline. Point to the `_datapath` created initially and specify the delimiter of the .csv file (,). Add the following code into the `Train()` method underneath the last step:
 
 ```csharp
-pipeline.Add(new TextLoader<TaxiTrip>(_datapath, useHeader: true, separator: ","));
+pipeline.Add(new TextLoader(_datapath).CreateFrom<TaxiTrip>(separator: ","));
```

Comment:
I think this needs to be (separator: ',') now, no?

Reply:
yes. Great catch!